src/99-appendices/08-coordinate-systems.md

```diff
@@ -151,6 +151,20 @@ Restricted keywords for the `<CoordSysType>CoordinateSystem` field in the
     [ACPC site](https://www.fieldtriptoolbox.org/faq/acpc/) on the FieldTrip
     toolbox wiki.
 
+-   `fsnative`: The subject-specific FreeSurfer T1w space. This corresponds to
+    the original scanner image after it is re-sampled and re-scaled
+    to an isotropic 1 mm 256x256x256 volume in RAS orientation. The origin
+    is wherever the original origin of the ``orig.mgz`` file is. One MUST
+    align the origin to the ``AC`` before using this coordinate system. If the origin is a random other location in the body, then it MUST be specified with ``Other`` and further details descriped in the ``CoordinateSystemDescription``.
```

Comment:
I suggest to not state what one must do (i.e. the action), but what the data should be like (the factual situation that results as outcome from the action). 

I also suggest not to mention in which cases `fsnative` is **not** to be used.

Reply:
That's a good point. I adjusted accordingly and took out some language.